components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -110,6 +114,12 @@ public class ServerConfigManagementService {
 
     private static final Log log = LogFactory.getLog(ServerConfigManagementService.class);
 
+    private static final List<String> FEDERATED_AUTHENTICATORS_TO_UPDATE = new ArrayList<>(Arrays.asList(
+            IdentityApplicationConstants.Authenticator.SAML2SSO.NAME,
+            IdentityApplicationConstants.Authenticator.PassiveSTS.NAME,
+            IdentityApplicationConstants.Authenticator.OIDC.NAME)
```

Comment:
We haven't changed the OIDC configs, right?

Reply:
Removed. Logic is updated to only change the required federated authenticator